src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx

```diff
@@ -267,8 +267,15 @@ If you've enabled the [Kubernetes events integration](/docs/integrations/kuberne
 
 ## Use the cluster explorer [#cluster-explorer-use]
 
-<Callout title="Deprecated">
-  The new Kubernetes Navigator UI is intended to replace the cluster explorer. However, you can enable the original cluster explorer experience at any time by turning off the Navigator toggle switch.
+<Callout
+  variant="important"
+  title="EOL NOTICE"
```

Comment:
```suggestion
  title="EOL notice"
```

Reply:
I've searched the entire site and I see that we always have it that way: `title="EOL NOTICE"`